client/client.js

```diff
@@ -0,0 +1,126 @@
+var log = require('minilog')('radar:client'),
+    Core = require('../core');
+
+function Client (name, id) {
```

Comment:
accountName should be a parameter here maybe. Also, we could do the keyGet here if we have it (since it's a concern of the instance)